mpas_analysis/ocean/climatology_map_waves.py

```diff
@@ -0,0 +1,526 @@
+        return obsDescriptor  # }}}
+
+    def build_observational_dataset(self, fileName):  # {{{
+        '''
+        read in the data sets for observations, and possibly rename some
+        variables and dimensions
+
+        Parameters
+        ----------
+        fileName : str
+            observation file name
+
+        Returns
+        -------
+        dsObs : ``xarray.Dataset``
+            The observational dataset
+        '''
+        # Authors
+        # -------
+        # Steven Brus
+
+        sectionName = self.taskName
+        climStartYear = self.config.getint(sectionName, 'era5ObsStartYear')
+        climEndYear = self.config.getint(sectionName, 'era5ObsEndYear')
+        timeStart = datetime.datetime(year=climStartYear, month=1, day=1)
+        timeEnd = datetime.datetime(year=climEndYear, month=12, day=31)
+
+        dsObs = xr.open_dataset(fileName)
+        dsObs = dsObs.rename({'time': 'Time'})
+        dsObs = dsObs.sel(Time=slice(timeStart, timeEnd))
+        dsObs.coords['month'] = dsObs['Time.month']
+        dsObs.coords['year'] = dsObs['Time.year']
+
+        dsObs = dsObs[self.variableList]
+
+        return dsObs  # }}}
+
+    # }}}
+
+
+class RemapSscciObservedWaveClimatology(RemapObservedClimatologySubtask):  # {{{
+    """
+    A subtask for reading and remapping SS_CCI wave observations
+    """
+    # Authors
+    # -------
+    # Steven Brus, Xylar Asay-Davis
+
+    def __init__(self, parentTask, seasons, fileName, outFilePrefix,
+                 variableList, subtaskName,
+                 comparisonGridNames=['latlon']):
+        # {{{
+        '''
+        Construct one analysis subtask for each plot (i.e. each season and
+        comparison grid) and a subtask for computing climatologies.
+
+        Parameters
+        ----------
+        parentTask :  ``AnalysisTask``
+            The parent (master) task for this subtask
+
+        seasons : list of str
+           A list of seasons (keys in ``constants.monthDictionary``) over
+           which the climatology should be computed.
+
+        fileName : str
+            The name of the observation file
+
+        outFilePrefix : str
+            The prefix in front of output files and mapping files, typically
+            the name of the field being remapped
+
+        variableList : list
+            List of observational variables to remap
+
+        comparisonGridNames : list of {'latlon', 'antarctic', 'arctic'},
+            optional
+            The name(s) of the comparison grid to use for remapping.
+
+        subtaskName : str, optional
+            The name of the subtask
+        '''
+        # Authors
+        # -------
+        # Steven Brus
+
+        self.variableList = variableList
+
+        # call the constructor from the base class (AnalysisTask)
+        super().__init__(parentTask, seasons, fileName, outFilePrefix,
+                         comparisonGridNames=['latlon'],
```

Comment:
Again, I think you want:
```suggestion
                         comparisonGridNames=comparisonGridNames,
```